src/scripts/ads/vpaid/VPAIDIntegrator.js

```diff
@@ -1066,6 +1073,15 @@ VPAIDIntegrator.prototype._trackError = function trackError(response, errorCode)
 	}
 };
 
+function showOriginalTimeDisplay(player, show) {
+  var children = player.controlBar.getChild('currentTimeDisplay').el_.children;
+  for (var i = 0; i < children.length; i++) {
+    if (children[i].id != 'bc_time_display') {
+      children[i].style.display = show ? '' : 'none';
```

Comment:
Why are you setting display to `''`? Shouldn't it be one of `'inline'`, `'block'` or `'inline-block'`?

Reply:
Setting display to '' resets the element's display property to the default.